ui/src/app/app.tsx

```diff
@@ -27,6 +30,7 @@ const routes: {
 } = {
     [workflowsUrl]: {component: workflows.component},
     [workflowTemplatesUrl]: {component: workflowTemplates.component},
+    [clusterWorkflowTemplatesUrl]: {component: clusterWorkflowTemplates.component},
```

Comment:
Should this be under the `WorkflowTempaltes` tab instead of its own tab? I'm not sure either way. Why do you think it should be its own?

Reply:
I posted my feedback on slack for UI changes. I had a discussion with @alexmt to have a tab on-page to switch between `workflow-template` and `cluster-workflow-template`. It requires a lot of refactoring on UI module. He suggested having a separate page will be a clean way and also it is easy to implement. I do agree with that. I don't want to overload the `workflow-template` page with multiple options.

Reply:
Good point. I agree with this.

Reply:
Good point, agreed.